.github/workflows/main.yml

```
@@ -1,65 +1,50 @@
+
+      - name: Run tests
+        env:
+          CHROMIUM_PATH: ${{ secrets.CHROMIUM_PATH || '' }}
+        run: npm run test:integration
+
+  build:
+    runs-on: ubuntu-latest
+    needs: lint-test
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.11)

[error] property "node-version" is not defined in object type {}

(expression)